src/main/java/view/NavigationMenu.java

```diff
@@ -0,0 +1,76 @@
+package view;
+
+import app.AppBuilder;
+import interface_adapter.login.LoginState;
+import interface_adapter.navigation.NavController;
+import interface_adapter.navigation.NavViewModel;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+/**
+ * Navigation menu to the application
+ */
+public class NavigationMenu extends JMenuBar implements ActionListener {
+
+    private final JButton buttonSearch = new JButton("Search");
+    private final JButton buttonRecipeCollection = new JButton("Recipe Collection");
+    private final JButton buttonChangePassword = new JButton("Change Password");
+    private final JButton buttonLogout = new JButton("Logout");
+    private NavController navController;
+
+    public NavigationMenu(NavViewModel navViewModel) {
+        this.add(buttonSearch);
+        this.add(buttonRecipeCollection);
+        this.add(buttonChangePassword);
+        this.add(buttonLogout);
+        this.setVisible(true);
+
+
+//        /**
+//         * Invoked when an action occurs.
+//         *
+//         * @param e the event to be processed
+//         */
+//        @Override
```

Comment:
I would recommend to add or commit to your version history. You can easily find it through previous versions without having to comment like this!